ttnn/cpp/ttnn/operations/ccl/reduce_scatter/reduce_scatter_pybind.cpp

```diff
@@ -29,15 +29,19 @@ void bind_reduce_scatter(pybind11::module& module, const ccl_operation_t& operat
                const uint32_t scatter_dim,
                ttnn::operations::reduction::ReduceType math_op,
                const uint32_t num_links,
-               const ttnn::MemoryConfig& memory_config) -> ttnn::Tensor {
-                return self(input_tensor, scatter_dim, math_op, num_links, memory_config);
+               const ttnn::MemoryConfig& memory_config,
+               const std::optional<size_t> num_workers,
+               const std::optional<size_t> num_buffers_per_channel) -> ttnn::Tensor {
+                return self(input_tensor, scatter_dim, math_op, num_links, memory_config, num_workers, num_buffers_per_channel);
             },
             py::arg("input_tensor"),
             py::arg("scatter_dim"),
             py::arg("math_op"),
             py::kw_only(),
             py::arg("num_links") = 1,
-            py::arg("memory_config") = std::nullopt});
+            py::arg("memory_config") = std::nullopt,
+            py::arg("num_workers") = std::nullopt,
+            py::arg("num_buffers_per_channel") = std::nullopt});
 }
```

Comment:
Are `num_links` and the two new args similar in nature? Should they all either have default values or be optional?

Reply:
They should be optional. The default value is set in the kernel